### PR TITLE
Feature/strategy type on asset

### PR DESCRIPTION
**Pull request by matthill8286**
# <JIRA issue / Title>

## TL;DR

- Added strategy type for playlist and asset tile

## Details

Checklist / Reminders:

-   [x] Documentation / Story is created / adapted
-   [x] Unit tests
-   [x] CSS changes are BEM-compliant

### CODE REVIEW

**Review by eTheMan80 — APPROVED** [review]

File: src/components/Organisms/AssetTile/AssetTile.interface.ts [hunk 1]
```diff
@@ -13,12 +13,19 @@ export type AssetTileLayout = 'auto' | 'listItem' | 'gridItem'
 
 export type AssetType = 'list' | 'collection' | 'disabled'
 
+export enum AssetStrategy {
```
Comment (philjonasfiltered) [inline src/components/Organisms/AssetTile/AssetTile.interface.ts:16]: the `strategy` prop is coming as a string from graphQL at the moment. When the playlist is not of the `structured` type, this prop is just an empty string. We may want to change this behaviour later on.
image

File: src/components/Organisms/AssetTile/AssetTile.tsx [hunk 2]
```diff
@@ -151,7 +152,7 @@ export const AssetTile: React.FC<AssetTileProps> = ({
                 opacity
               />
             )}
-            {showLock && (
+            {showLock && !strategyType && (
```
Comment (philjonasfiltered) [inline src/components/Organisms/AssetTile/AssetTile.tsx:155]: Are all strategy types behind a lock? Wouldn't it be less ambiguous for the `timebox` type to have a clock icon? Not a blocker at all, just a question.

Comment (matthill8286) [reply src/components/Organisms/AssetTile/AssetTile.tsx:155]: No, that says  if showLock if passed in which it's optional and Not a strategy, as no strategy is behind a lock. Why would I be setting the lock functionality to a clock icon?? That is totally separate to locking as that a landing page only feature.

File: src/components/Organisms/AssetTile/AssetTile.interface.ts (hunk 1 above)
Comment (philjonasfiltered) [reply src/components/Organisms/AssetTile/AssetTile.interface.ts:16]: @matthill8286 Not sure. But I double checked in Postman as well:
image

**Review by philjonasfiltered — APPROVED** [review]

**Pull request merged**